qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -468,11 +563,12 @@ def ramp(self, mode: str="safe") -> None:
             if slave.field_target() != cur:
                 if slave.field_ramp_rate() == 0:
                     raise ValueError(f'Can not ramp {slave}; ramp rate set to'
-                                     ' zero!')
+                                        ' zero!')
 
         # then the actual ramp
         {'simul': self._ramp_simultaneously,
-         'safe': self._ramp_safely}[mode]()
+            'safe': self._ramp_safely,
+            'simul_block':self._ramp_simultaneously_blocking}[mode]()
```

Comment:
there is some weird excessive indentation in the whole commit, could this be fixed for readability?